CppCoreGuidelines.md

```diff
@@ -19031,6 +19032,30 @@ Failing to follow this results in difficult to diagnose errors for clients of a
 
 A test should verify that the header file itself compiles or that a cpp file which only includes the header file compiles.
 
+### <a name="Rs-incscope"></a>SF.12: Use the quoted or angle bracket form of `#include` to distinguish standard headers from others
+
+##### Reason
+
+To understand dependencies it is important to know where included files come from. Given the different search algorithms, 
+the angle form (`<>`) identifies standard headers, the quoted (`""`) everything else.
```

Comment:
This contradicts the note I mentioned:

> in general programmers should use the < > form for headers provided with the implementation, and the " " form for sources outside the control of the implementation.

That strongly suggests that angle brackets should be used for non-standard headers "provided with the implementation" such as `Windows.h` and `unistd.h`, or system libraries like `zlib.h` or `Python.h`. Limiting it to just standard headers is too restrictive, and ignores the "in between" cases that are not standard headers but also not one of the project's own source files.

I have no idea how Windows does things, but on many POSIX systems it doesn't make sense to do `#include "X11/Xlib.h"` with quotes but `#include <stdio.h>` with angle brackets, when both are likely to be found in `/usr/include`.

Reply:
This is still too restrictive. It's often appropriate to use `<>` to include headers from within the same project.